conf/deployment/gcp/ipi_1az_rhcos_3m_3w.yaml

```diff
@@ -0,0 +1,8 @@
+---
+ENV_DATA:
+  platform: 'gcp'
+  deployment_type: 'ipi'
+  worker_replicas: 3
+  master_replicas: 3
+  master_instance_type: 'n1-standard-16'
+  worker_instance_type: 'n1-standard-16'
```

Comment:
are we using same instance type for master and worker?

Reply:
As of now, Yes!